drivers/dai/nxp/sai/sai.c

```diff
@@ -353,6 +353,68 @@ static int sai_config_set(const struct device *dev,
 	 * different than the hardware rate (a.k.a this one).
 	 */
 	data->cfg.rate = bespoke->fsync_rate;
+
+#ifdef CONFIG_SAI_IMX93_ERRATA_051421
+	/* a few notes about the following piece of code:
+	 *
```

Comment:
This looks very good. What do you think about adding a function in order to improve readability of sai_config_set?

e.g 
```
#ifdef CONFIG_SAI_IMX93_ERRATA_051421

void sai_config_set_errata_05152(struct *device...)

{ 
//code for errata
}

#else

void void sai_config_set_errata_05152(struct *device...)
{
///
}

#endif

```